content/en/news/release-notes.md

```diff
@@ -6,6 +6,14 @@ weight: 1
 
 For additional information check our [sprint demo videos](https://www.youtube.com/channel/UCcm2NzDN_UCZKk2yYmOpc5w) and [blogs](https://medium.com/kialiproject).
 
+Upgrade Change Notes:
+
+The following fields are no longer used by the Kiali CR and may be safely removed, if set. Leaving them in an existing CR will not be a problem, but they are unused.
```

Comment:
This likely needs to be re-worded if we get the CRD Schema merged along with this stuff. If that happens, then it is no longer true that leaving them in will not be a problem. It will be a problem - the CR will fail schema validation and the Kiali CR will be rejected.

We should say something like, "These are no longer used and must be removed."